apps/dotcdn/src/app/app.component.html

```diff
@@ -0,0 +1,88 @@
+                        height="1.5rem"
+                        *ngIf="VM.isChartLoading; else valuesBlock"
+                    ></p-skeleton>
+                    <ng-template #valuesBlock>
+                        <h3 class="dot-cdn__stats-value">{{ stats.value }}</h3>
+                    </ng-template>
+                    <dot-icon [name]="stats.icon" size="32"></dot-icon>
+                </div>
+            </div>
+
+            <div class="dot-cdn__chart">
+                <dot-spinner
+                    [size]="'4rem'"
+                    [borderSize]="'3px'"
+                    *ngIf="VM.isChartLoading; else chartBlock"
+                    [ngStyle]="{ height: chartHeight }"
+                ></dot-spinner>
+                <ng-template #chartBlock>
+                    <h3 class="dot-cdn__tab-content-label">Bandwidth Served</h3>
+                    <p-chart
+                        #chart
+                        type="line"
+                        [data]="VM.chartData"
+                        width="100%"
+                        [height]="chartHeight"
+                        [options]="options"
+                    ></p-chart>
+                </ng-template>
+            </div>
+        </div>
+    </p-tabPanel>
+    <p-tabPanel header="Flush Cache" *ngIf="vmPurgeLoaders$ | async as VMPurgeLoaders">
+        <div class="dot-cdn__tab-content--contained">
+            <form [formGroup]="purgeZoneForm">
+                <div class="dot-cdn__tab-content__row">
+                    <h3 class="dot-cdn__tab-content-label">Purge URL List</h3>
+                    <p>
+                        Purging an url list will remove the file from the CDN cache and re-download
+                        it from your server. Please enter the exact CDN URL of each individual file.
+                        Wildcard values are not supported.
+                    </p>
+                    <textarea
+                        [rows]="5"
+                        pInputTextarea
+                        formControlName="purgeUrlsTextArea"
+                    ></textarea>
+                    <button
+                        pButton
+                        [label]="VMPurgeLoaders.isPurgeUrlsLoading ? 'loading...' : 'Purge URL'"
+                        [disabled]="purgeZoneForm.get('purgeUrlsTextArea').value.length === 0"
+                        (click)="purgeUrls()"
+                    ></button>
+                </div>
+            </form>
+            <div class="dot-cdn__tab-content__row">
+                <h3 class="dot-cdn__tab-content-label">Purge Pull Zone</h3>
+                <p>
+                    Purging a Pull Zone will remove completely all of its files from cache and foce
+                    everything to be re-downloaded from your origin. If your zone uses a lot of
+                    storage, this ight send a large amount of traffic back to your server, so use
+                    with caution.
```

Comment:
```suggestion
                    Purging a Pull Zone will altogether remove all of its files from the cache and force
                    everything to be re-downloaded from your origin. If your zone uses a lot of storage,
                    this might send a large amount of traffic back to your server, so use with caution.
```